float64 temperature
float64 freeMemory
float64 percentFreeMemory
float64 percentCPUUsage
float64 ping
bool underVoltage
bool GPUCoreState
bool SDRAMState
string fatalComponent
GPIOStatus[] GPIOState

